### ISSUE TITLE
Use Meson builtins instead of building path strings manually

### ISSUE THREAD

**Issue by jpakkane** [winksaville/sadie#85]
A lot of the files build strings manually, like this:

```
runtimeIncDirs += include_directories(
  '@0@/incs'.format(meson.current_source_dir())
)

runtimeSrcs += [
  '@0@/srcs/ac_pci.c'.format(meson.current_source_dir()),
  '@0@/srcs/ac_pci_print.c'.format(meson.current_source_dir()),
]
```

This is a lot of typing and also error prone. Meson provides builtins so people don't have to do this. The native way is the following:

```
runtimeIncDirs += [include_directories('.')]
runtimeSrcs += files('srcs/ac_pci.c', 'srcs/ac_pci_print.c')
```

Meson remembers the paths in `include_directories` and `files´. You can use them anywhere and it will do the proper path expansion for you in the background.

**Comment by winksaville** [comment]
I'll fix those, txs. Some I do need, like the ones for tools which aren't
in the path or files which are lower down in the hierarchy, correct?

**Comment by jpakkane** [comment]
Not really. If you use `find_program` and `files` you can use the result anywhere and it will do the right thing. The only limitation is that you can't refer to things that are defined in "the future". That is if you first subdir from top level into `a` and then to `b` you can use any declaration made in `a` in `b` but not the other way around (because `b` has not been visited so the things created there do not exist yet).

**Comment by winksaville** [comment]
OK I'll use find_xxx more.

**Comment by winksaville** [comment]
@jpakkane, could you add 0.35.0-dev to pip3 so I can use the latest code now and not have to wait for 0.35.0 release? Here is what I see looking at the available versions in pip3 now:

```
$ pip3 install meson==
Collecting meson==
  Could not find a version that satisfies the requirement meson== (from versions: 0.29.0.dev1, 0.29.0.dev2, 0.29.0.dev3, 0.29.0, 0.30.0, 0.31.0, 0.32.0, 0.33.0, 0.34.0)
No matching distribution found for meson==
```
